src/language-js/print/assignment.js

```diff
@@ -205,7 +205,7 @@ function shouldBreakAfterOperator(path, options, print, hasShortKey) {
   let node = rightNode;
   const propertiesForPath = [];
   for (;;) {
-    if (node.type === "UnaryExpression") {
+    if (node.type === "UnaryExpression" || node.type === "AwaitExpression") {
```

Comment:
If we are doing this, I think we should treat `yield` the sameway, though it's not very common to be on RHS.

Reply:
Ah, I agree with you,thank you.
Maybe `typeof` too？